flink-connectors/flink-connector-files/src/test/java/org/apache/flink/connector/file/src/FileSourceTextLinesITCase.java

```diff
@@ -126,8 +127,9 @@ private void testBoundedTextFileSource(
                         .build();
 
         final StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+        env.setRuntimeMode(RuntimeExecutionMode.BATCH);
```

Comment:
Can we add a new test to verify dynamic parallelism inference for file sources?
We may use `miniCluster.getExecutionGraph(jobId)` to get the execution graph of the job and check the parallelism of the source.